examples/cu_consolemon/src/sysinfo.rs

```diff
@@ -0,0 +1,220 @@
+use libmacchina::{
+    traits::GeneralReadout as _, traits::KernelReadout as _, traits::MemoryReadout as _,
+    traits::PackageReadout as _, GeneralReadout, KernelReadout, MemoryReadout, PackageReadout,
+};
+
+use pfetch_logo_parser::{Color, Logo, LogoPart};
+use std::{env, fmt::Display, str::FromStr};
+
+#[derive(Debug, PartialEq)]
+pub enum PfetchInfo {
+    Ascii,
+    Title,
+    Os,
+    Host,
+    Kernel,
+    Uptime,
+    Cpu,
+    Memory,
+    Shell,
+    Editor,
+    Wm,
+    De,
+    Palette,
+    BlankLine,
+}
+
+impl Display for PfetchInfo {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", format!("{self:?}").to_lowercase())
+    }
+}
+
+impl FromStr for PfetchInfo {
```

Comment:
The [https://docs.rs/strum/latest/strum/](strum) crate has a lot of this "enum to string or string to enum" behavior built in. It's pretty light and probably worth bringing it in as a dependency just for it's usefulness there. Like you can just put a `#[derive(strum::AsRefStr)]` above the enum you want string conversions for and it will generate all the conversions for you. Check it out!